README.rst

```diff
@@ -75,8 +76,8 @@ then you can install from there with::
 Install from GitHub
 ~~~~~~~~~~~~~~~~~~~
 
-Work on rst2pdf has restarted on GitHub, with the goals of supporting
-Python 3, addressing outstanding issues, and not breaking anything. You
+Work on rst2pdf has restarted on GitHub, with the goals of adding new
+features, addressing outstanding issues, and not breaking anything. You
 can clone the repository and install this version::
 
     $ git clone https://github.com/rst2pdf/rst2pdf rst2pdf
```

Comment:
Unrelated, but you don't need the second argument to `git clone` - it will automatically clone to a directory with the same name as the repo.

Reply:
Addressed.